common/src/main/java/com/thekillerbunny/goofyplugin/lua/GoofyAPI.java

```diff
@@ -262,6 +263,28 @@ public void loadAvatar(@LuaNotNil String playerUUID) {
         NetworkStuff.getUser(new UserData(uuid));
     }
 
+    @LuaWhitelist
+    @LuaMethodDoc(
+        overloads = {
+            @LuaMethodOverload(
+                argumentTypes = {String.class},
+                argumentNames = {"playerUUID"}
+            )
+        },
+        value = "goofy.get_player_color"
+    )
+    public String getPlayerColor(String playerUUID) {
+        UUID uuid = UUID.fromString(playerUUID);
+        if (AvatarManager.getAvatarForPlayer(uuid) == null) {
+            return "#000000";
+        }
+        Avatar avatar = AvatarManager.getLoadedAvatar(uuid);
+        if (avatar == null) {
+            return "#000000";
```

Comment:
as above
```suggestion
            return null;
```

Reply:
it's important to be able to distinguish between invalid avatars and actual black